solver/jobs.go

```diff
@@ -53,6 +54,7 @@ type state struct {
 
 	vtx          Vertex
 	clientVertex client.Vertex
+	origDigest   digest.Digest // original LLB digest. TODO: probably better to use string ID so this isn't needed
```

Comment:
TODO will be done in another PR?

Reply:
Yes, this is a solver internals change that I don't want to do atm.